modules/ROOT/pages/flex-gateway-apis.adoc

```diff
@@ -0,0 +1,22 @@
+= APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To check basic information about the APIs associated with your flex gateways from Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the left menu.
+. Click the name of the gateway whose APIs you want to check.
+. Click *APIs*.
++
+For each API, you will see the name, version, instance ID, label, and the date on which it was added.
```

Comment:
you **will** see the name > Runtime Manager shows the name... 
or > the UI shows the name...

Reply:
Applied